packages/migrations/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { runV1MigrationsAsync } from './v1/migration';
 export { runV2MigrationsAsync } from './v2/migration';
+export { ArtifactWriter } from './artifact_writer';
```

Comment:
Could you explain - why do we need to use ArtifactWriter outside of migrations?

Reply:
I can imagine people writing their own migrations wanting access to this method... Any hesitations about exposing it?

Reply:
No. Let's just make sure that it's ready (documented and has nice API).

Reply:
Good point. Let's punt on this for now.